UnityProject/Assets/Scripts/Managers/AdminCommandsManager.cs

```diff
@@ -559,6 +559,12 @@ public void CmdChangeBudget(int budget, NetworkConnectionToClient sender = null)
 			CargoManager.Instance.Credits = budget;
 		}
 
+		[Command(requiresAuthority = false)]
+		public void CmdChangeCargoConnectionStatus(bool status, NetworkConnectionToClient sender = null)
```

Comment:
PR looks good. My only nitpick is this method name and the variable that receives. It isn't entirely intuitive to think that sending ``true`` to ``ChangeCargoConnectionStatus`` will make it be offline. I would rethink the naming so it makes more sense.

Reply:
changed